tetris.py

```diff
@@ -285,7 +228,6 @@ def current_block_mover(self):
                 )
             )
             self.full_line_clear()
```

Comment:
Change `current_block_mover()` to return `True` when a new block should be added:

```suggestion
            self.full_line_clear()
            return True   # new block needed
```